bedrock/products/templates/products/vpn/includes/macros.html

```diff
@@ -54,6 +54,26 @@ <h2 class="vpn-hero-sub-heading">{{ subheading }}</h2>
 </section>
 {%- endmacro %}
 
+{% macro vpn_platform_hero(heading, subheading, desc, image_url) -%}
```

Comment:
Created a new marco that can define a single image (which vary between desktop and mobile platform pages)